### PR TITLE
Not same people two years in row

### PR DESCRIPTION
**Pull request by skurpi**
Added check so that people don't get the same person two years in a row.

I forgot to do this when I did the changes, so I hope they still work, assuming you haven't had time to change too much.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,12 @@ Intro
 **secret-santa** can help you manage a list of secret santa participants by
 randomly assigning pairings and sending emails. It can avoid pairing 
 couples to their significant other, and allows custom email messages to be 
-specified.
+specified. If this is not your first year then it will also make sure that
+people don't get the same people two years in a row.
+
+This project is forked off from underbluewaters who has done most of the
```
Comment (skurpi) [inline README.md]: You can definitely skip these rows

File: README.md [hunk 2]
```diff
@@ -63,7 +76,7 @@ Here is the example configuration unchanged:
 
       Nothing could possibly go wrong...
 
-      http://github.com/underbluewaters/secret-santa
+      http://github.com/skurpi/secret-santa
```
Comment (skurpi) [inline README.md]: and skip this as well

**Comment by skurpi** [comment]
Now that I learned how pull requests actually work, I cleaned my branch from stealing credit and just containing the real features.

**Comment by felixvd** [comment]
Is this not covered by DONT-PAIR already?

**Comment by skurpi** [comment]
You're right @felixvd

**Comment by skurpi** [comment]
@felixvd Actually it's not. DONT-PAIR will make sure that two people cannot get each other. While this pull requests enables the scenario of person A getting person B as their santee, but not the other way around.

**Pull request closed without merging**